jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/AbstractRuntimeManager.java

```diff
@@ -87,7 +87,7 @@
  */
 public abstract class AbstractRuntimeManager implements InternalRuntimeManager {
     
-    private static final Logger logger = LoggerFactory.getLogger(AbstractRuntimeManager.class);
+    protected static final Logger logger = LoggerFactory.getLogger(AbstractRuntimeManager.class);
```

Comment:
I think this should be kept as it was - `logger` for PerRequestRuntimeManager should be created in its own class and not inherit this to avoid any issues when configuring logging.

Reply:
Kept it back to the previous qualifier, thanks